qupath-gui-fx/src/main/java/qupath/lib/gui/JavadocViewerRunner.java

```diff
@@ -62,6 +62,8 @@ public class JavadocViewerRunner implements Runnable {
     private static final StringProperty javadocPath = PathPrefs.createPersistentPreference(JAVADOC_PATH_PREFERENCE, null);
     private final JavadocViewerCommand command;
 
+    private static final String JAVADOCS_STYLE_CLASS = "javadoc-viewer";
```

Comment:
Could you move that line above `private final JavadocViewerCommand command;`? I prefer defining constants above everything else